static/js/around-map.js

```diff
@@ -20,10 +20,14 @@ worker.addEventListener('message', response => {
     el.classList.add('marker')
     el.id = poi._id
 
-    const popup = new mapboxgl.Popup({offset: [10, 0]})
-      .setHTML(`<h2>${poi.name}</h2>
-                <h3>Comments:</h3>
-                <p>${poi.comments}</p>`)
+    let popup = new mapboxgl.Popup({offset: [10, 0]})
+    if(poi.comments.length === 0 || poi.comments[0] == [""]) {
```

Comment:
/info: I delete the popup (but this code is still existant in a new component name `Comments`